<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HelloAnyChat</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>259</y>
     <width>201</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="UserListhorizontalLayout">
    <item>
     <widget class="QListWidget" name="UserlistWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>511</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="RemoteUserhorizontalLayout">
    <item>
     <widget class="QLabel" name="RemoteUserlabel">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>RemoteUser</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>201</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="LocalUserhorizontalLayout">
    <item>
     <widget class="QLabel" name="LocalUserlabel">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>LocalUser</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QProgressBar" name="LocalUserProgressBar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>600</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="RemoteUserprogressBar">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>390</y>
     <width>541</width>
     <height>16</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>279</x>
     <y>419</y>
     <width>511</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="SysMsgTextEdit"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLineEdit" name="SendMsglineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="SendMsg_Btn">
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="UserListlabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Online Remote User List</string>
   </property>
  </widget>
  <widget class="QLabel" name="localAudioBpslabel">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>370</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="localVideoBpslabel">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>412</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="RemoteAudioBpslabel">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>490</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="RemoteVideoBpslabel">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>530</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ServerAddrlable">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>服务器:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ServerAddrlineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>69</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="ServerPortlabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>端  口:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ServerPortlineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="RoomIdlabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>房间ID:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="RoomId_lineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="AppIdlabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>应用ID:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Appid_lineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="Appid_checkBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>125</y>
     <width>20</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Login_btn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLabel" name="UserNamelabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>29</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="UserNamelineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>36</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
